internal/config/files.go

```diff
@@ -3,10 +3,11 @@ package config
 import "strings"
 
 const (
-	SubFiles       string = "{files}"
-	SubAllFiles    string = "{all_files}"
-	SubStagedFiles string = "{staged_files}"
-	PushFiles      string = "{push_files}"
+	SubFiles                      string = "{files}"
+	SubAllFiles                   string = "{all_files}"
+	SubAllFilesIncludingUntracked string = "{all_including_untracked_files}"
```

Comment:
@sanmai-NL , do you really need this template? Why can't a custom `{files}` be used instead?